client/src/components/NationwideMap.vue

```diff
@@ -112,21 +128,36 @@ export default defineComponent({
     /**
      * Sets the visibility of the layer with the given styleLayerId.
      */
-    setDataLayerVisibility(styleLayerId: string, visible: boolean): void {
+    setDataLayerVisibility(layerId: string): void {
```

Comment:
lets change this to setDataLayerVisible if it just sets layerId visible now

Reply:
Done and combined with toggle data layer since it really only had one extra line